onnxruntime/test/server/integration_tests/function_tests.py

```diff
@@ -330,24 +341,29 @@ class HttpEndpointTests(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cmd = [cls.server_app_path, '--http_port', str(cls.server_port), '--model_path', os.path.join(cls.model_path, 'mnist.onnx'), '--log_level', cls.log_level]
-        print('Launching server app: [{0}]'.format(' '.join(cmd)))
+        onnx_model = os.path.join(cls.model_path, 'mnist.onnx')
```

Comment:
A lot of code is duplicated here. Perhaps pull at least the URL to one single location to make it easy to change